File: Payhub - SDK/java_SDK/PayHubSDK/src/main/java/com/payhub/ws/vt/WebhookConfiguration.java
package com.payhub.ws.vt;

import com.fasterxml.jackson.annotation.JsonIgnore;
import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.core.JsonProcessingException;
import com.fasterxml.jackson.databind.DeserializationFeature;
import com.fasterxml.jackson.databind.ObjectMapper;

public class WebhookConfiguration{
	public static final String WEBHOOK_LINK = "adminSettings/webhookConfiguration";

    private String endPoint;

    private Boolean mobileHub;

    private Boolean recurringBill;

    private Boolean virtualHub;

    private Boolean webhookActive;

    private Boolean api;

    private Boolean batchIsActive;

	/**
	 * @return the endPoint
	 */
	public String getEndPoint() {
		return endPoint;
	}

	/**
	 * @param endPoint the endPoint to set
	 */
	public void setEndPoint(String endPoint) {
		this.endPoint = endPoint;
	}

	/**
	 * @return the mobileHub
	 */ 
	public Boolean getMobileHub() {
		return mobileHub;
	}

	/**
	 * @param mobileHub the mobileHub to set
	 */
	public void setMobileHub(Boolean mobileHub) {
		this.mobileHub = mobileHub;
	}

	/**
	 * @return the recurringBill
	 */ 
	public Boolean getRecurringBill() {
		return recurringBill;
	}

	/**
	 * @param recurringBill the recurringBill to set
	 */
	public void setRecurringBill(Boolean recurringBill) {
		this.recurringBill = recurringBill;
	}
	
	/**
	 * @return the virtualHub
	 */
	public Boolean getVirtualHub() {
		return virtualHub;
	}

	/**
	 * @param virtualHub the virtualHub to set
	 */
	public void setVirtualHub(Boolean virtualHub) {
		this.virtualHub = virtualHub;
	}

	/**
	 * @return the webhookActive
	 */  
	public Boolean getWebhookActive() {
		return webhookActive;
	}

	/**
	 * @param webhookActive the webhookActive to set
	 */
	public void setWebhookActive(Boolean webhookActive) {
		this.webhookActive = webhookActive;
	}

	/**
	 * @return the api
	 */  
	public Boolean getApi() {
		return api;
	}

	/**
	 * @param api the api to set
	 */
	public void setApi(Boolean api) {
		this.api = api;
	}

	/**
	 * @return the batchIsActive
	 */
	public Boolean getBatchIsActive() {
		return batchIsActive;
	}

	/**
	 * @param batchIsActive the batchIsActive to set
	 */
	public void setBatchIsActive(Boolean batchIsActive) {
		this.batchIsActive = batchIsActive;
	}
    
	public String toString(){
		ObjectMapper mapper = new ObjectMapper();
        mapper.disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES);
        String jsonInString;
		try {
			jsonInString = mapper.writerWithDefaultPrettyPrinter().writeValueAsString(this);
			return jsonInString;
		} catch (JsonProcessingException e) {
			// TODO Auto-generated catch block
			return null;
		}
        
	}
    
}

File: Payhub - SDK/php_SDK/com/payhub/ws/vt/WebhookConfiguration.php
<?php

/**
 * Created by PhpStorm.
 * User: agustin
 * Date: 12/18/15
 * Time: 1:33 PM
 */
class WebhookConfiguration
{
    public static $WEBHOOK_LINK = "adminSettings/webhookConfiguration";
    public $endPoint;
    public $mobileHub;
    public $recurringBill;
    public $virtualHub;
    public $webhookActive;
    public $api;
    public $batchIsActive;

    /**
     * WebhookConfiguration constructor.
     */
    public function __construct()
    {
    }

    /**
     * @return mixed
     */
    public function getEndPoint()
    {
        return $this->endPoint;
    }

    /**
     * @param mixed $endPoint
     */
    public function setEndPoint($endPoint)
    {
        $this->endPoint = $endPoint;
    }

    /**
     * @return mixed
     */
    public function getMobileHub()
    {
        return $this->mobileHub;
    }

    /**
     * @param mixed $mobileHub
     */
    public function setMobileHub($mobileHub)
    {
        $this->mobileHub = $mobileHub;
    }

    /**
     * @return mixed
     */
    public function getRecurringBill()
    {
        return $this->recurringBill;
    }

    /**
     * @param mixed $recurringBill
     */
    public function setRecurringBill($recurringBill)
    {
        $this->recurringBill = $recurringBill;
    }

    /**
     * @return mixed
     */
    public function getVirtualHub()
    {
        return $this->virtualHub;
    }

    /**
     * @param mixed $virtualHub
     */
    public function setVirtualHub($virtualHub)
    {
        $this->virtualHub = $virtualHub;
    }

    /**
     * @return mixed
     */
    public function getWebhookActive()
    {
        return $this->webhookActive;
    }

    /**
     * @param mixed $webhookActive
     */
    public function setWebhookActive($webhookActive)
    {
        $this->webhookActive = $webhookActive;
    }

    /**
     * @return mixed
     */
    public function getApi()
    {
        return $this->api;
    }

    /**
     * @param mixed $api
     */
    public function setApi($api)
    {
        $this->api = $api;
    }

    /**
     * @return mixed
     */
    public function getBatchIsActive()
    {
        return $this->batchIsActive;
    }

    /**
     * @param mixed $batchIsActive
     */
    public function setBatchIsActive($batchIsActive)
    {
        $this->batchIsActive = $batchIsActive;
    }


    public static function fromArray($data)
    {
        $wh = new WebhookConfiguration();
        foreach ($data as $key => $value) {
            if (property_exists(get_class($wh), $key)) {
                    $wh->{$key} = $value;
            }
        }
        return $wh;
    }

    public function object_unset_nulls_for_send()
    {
        $object = (object) array_filter((array) $this, function ($val) {
            return !is_null($val);
        });
        return $object;
    }

}